Misc/NEWS.d/next/Library/2020-09-23-21-58-34.bpo-40066.f1dr_5.rst

```diff
@@ -0,0 +1,4 @@
+Enum's `repr()` and `str()` have changed: `repr()` is now *EnumClass.MemberName*
+and `str()` is *MemberName*.  Additionally, stdlib Enum's whose contents are
+available as module attributes, such as `RegexFlag.IGNORECASE`, have their
+`repr()` as *module.name*, e.g. `re.IGNORECASE`.
```

Comment:
Maybe also add a section to what's new in 3.10?

Reply:
I thought `blurb` did that?

Reply:
No, blurb just updates Misc/NEWS, which is an endless list of things (though more focused than the commit log :-).

What's new in 3.10 is a separate doc written by human editor who curate the most important news and group it by affected area.

Reply:
https://docs.python.org/3.10/whatsnew/3.10.html

Reply:
Also the `versionchanged` directive is needed in the module documentation.